core/vtk/ttkTriangulationRequest/ttkTriangulationRequest.cpp

```diff
@@ -1,24 +1,44 @@
+#include <ttkMacros.h>
 #include <ttkTriangulationRequest.h>
+#include <ttkUtils.h>
 
 using namespace std;
 using namespace ttk;
 
 vtkStandardNewMacro(ttkTriangulationRequest);
-int ttkTriangulationRequest::doIt(vector<vtkDataSet *> &inputs,
-                                  vector<vtkDataSet *> &outputs) {
 
-  Memory m;
+int ttkTriangulationRequest::FillInputPortInformation(int port,
+                                                      vtkInformation *info) {
+  if(port == 0) {
+    info->Set(vtkAlgorithm::INPUT_REQUIRED_DATA_TYPE(), "vtkDataSet");
+    return 1;
+  }
+  return 0;
+}
 
-  vtkDataSet *input = inputs[0];
-  vtkUnstructuredGrid *output = vtkUnstructuredGrid::SafeDownCast(outputs[0]);
+int ttkTriangulationRequest::FillOutputPortInformation(int port,
+                                                       vtkInformation *info) {
+  if(port == 0) {
+    info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkUnstructuredGrid");
+    return 1;
+  }
+  return 0;
+}
 
-  Triangulation *triangulation = ttkTriangulation::getTriangulation(input);
+int ttkTriangulationRequest::RequestData(vtkInformation *request,
+                                         vtkInformationVector **inputVector,
+                                         vtkInformationVector *outputVector) {
+
+  Memory m;
```

Comment:
I'd recommend to remove memory trackers.
they're based on the proc filesystem stats, which is only supported on linux and which provides very imprecise memory usage estimations.